core/src/main/java/org/apache/accumulo/core/conf/PropertyType.java

```diff
@@ -117,6 +121,7 @@ String getFormatDescription() {
    * @return true if value is valid or null, or if this type has no regex
    */
   public boolean isValidFormat(String value) {
+    Preconditions.checkState(predicate != null, "Predicate was null, maybe this enum was serialized????");
```

Comment:
For some reason the changes I made resulted in findbugs complaining about code I didn't change.  These changes were a result of a findbugs complaint.